.github/workflows/test-kurtosis-assertoor.yml

```diff
@@ -4,35 +4,45 @@ on:
   schedule:
     - cron: "0 2,14 * * *" # runs at 2am and 2pm UTC
   workflow_dispatch:
-  # pull_request:
-  #   branches:
-  #     - sudeep/kurtosis_assertoor
-  #   types:
-  #     - opened
-  #     - reopened
-  #     - synchronize
-  #     - ready_for_review
-  # push:
-  #   branches:
-  #     - sudeep/kurtosis_assertoor
+  pull_request:
+    branches:
+      - kurtosis_ci_fix
```

Comment:
Shouldn't this be `main`?

Reply:
oh this portion is just meant for testing, and has to be commented out before merging; please check now